gamedata/alldefs_post.lua

```diff
@@ -237,6 +237,32 @@ function UnitDef_Post(name, uDef)
 			end
 		end
 
+		if modOptions.unbacom then	
+			if uDef.customparams.isunbacom or uDef.customparams.iscommander then
+				uDef.customparams.evolution_timer = modOptions.unbacomleveluprate*60
+			end
+			if  name == "armcom" then
+				uDef.customparams.evolution_announcement = "Armada commanders have upgraded to level 2"
```

Comment:
Since all the commanders that haven't perished and been revived would level up at the same time, would it not be better to have a more general message used by all 3?

Reply:
There's an extra parameter in the evolution gadget that specifies "combat range" which prevents evolution if a hostile unit is in the radius.

This causes desynchronization of evolutions. Getting a message "Armada Commander upgraded" kinda helps players parse relevant info 

I was considering just removing evolution messages because they're so numerous. I personally kinda like being notified, but it does seem rather janky. 

Opinion?